docs/content/getting-started/overview.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Overview | Dagster Docs"
```

Comment:
Thoughts on naming this something like "Why Dagster?" or "Dagster: What and Why"? The existing landing page satisfies at least some definition of an Overview, and this page is doing something a little more specific.

Reply:
Renamed and updated in the sidenav. This feels a bit weird being right above "Hello, Dagster!" in the sidenav, but that might just be me.